spec/unit/coinbase/webhook_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+describe Coinbase::Webhook do
+  let(:api_client) { double('api_client') }
+  let(:webhooks_api) { double('Coinbase::Client::WebhooksApi') }
+  let(:webhook_model) do
+    Coinbase::Client::Webhook.new(
+      id: 'webhook_id',
+      network_id: 'base-mainnet',
+      event_type: 'erc20_transfer',
+      event_filters: [{ 'contract_address' => '0x833589fcd6edb6e08f4c7c32d4f71b54bda02913' }],
```

Comment:
Is this not returned as a `Coinbase::Client::WebhookEventFilter`?

When you're testing this in console with live data, is this what is being returned?

Reply:
yes it is being returned:
```
[3] pry(main)> tmp = Coinbase::Webhook.list.first
=> Coinbase::Webhook{id: '66b2ac80e8abf9381a162137', network_id: 'base-mainnet', event_type: 'erc20_transfer', notification_uri: 'https://update', event_filters: '[{"contract_address":"0x833589fcd6edb6e08f4c7c32d4f71b54bda02913"}]'}
```